src/tm2/tm-device.h

```diff
@@ -95,9 +95,45 @@ namespace librealsense
         void set_manual_exposure(bool manual);
 
         // Pose interfaces
+
+        /**
+         * Get relocalization map from T265
+         * \param[out] lmap_buf map data as a binary blob
+         * \return true if success
+         */
         bool export_relocalization_map(std::vector<uint8_t>& lmap_buf) const override;
+
+        /**
+         * Send relocalization map to T265. Only one relocalization map can be imported.
```

Comment:
```suggestion
         * Load relocalization map onto T265. Only one relocalization map can be imported at a time; 
         * any previously existing map will be overwritten.
```
Again, not sure these statements are correct, so please fix if needed. I see lower in documentation a reference to "current map" and "imported map", so if these can exist simultaneously, this function is a probably a good place to define and explain those two maps.